bin/pybsn-schema

```diff
@@ -43,7 +46,7 @@ def pretty_type(node):
     t = node['typeSchemaNode']
 
     if t['leafType'] == 'ENUMERATION':
-        names = [x for x in t['typeValidator'] if x['type'] == 'ENUMERATION_VALIDATOR'][0]['names'].keys()
+        names = list([x for x in t['typeValidator'] if x['type'] == 'ENUMERATION_VALIDATOR'][0]['names'].keys())
```

Comment:
python 3 doesn't support list comprehensions?

Reply:
It does... what it doesn't support is subscripts on the specialized `dict_keys` collection that's returned from `<dict>.keys()`:
```
    names = names[:4] + ['...']
TypeError: 'dict_keys' object is not subscriptable
```

Thus converting the `.keys()` result explicitly to a list.

Reply:
ah, i see ... 
> in Python 3, the dict.keys() method returns a dictionary view object,